File: spring-data-requery/src/test/java/org/springframework/data/requery/repository/sample/SampleConfig.java
package org.springframework.data.requery.repository.sample;

import org.springframework.context.annotation.Configuration;
import org.springframework.data.requery.configs.RequeryTestConfiguration;
import org.springframework.data.requery.repository.config.EnableRequeryRepositories;

@Configuration
@EnableRequeryRepositories
public class SampleConfig extends RequeryTestConfiguration {
}

File: spring-data-requery-kotlin/src/test/kotlin/org/springframework/data/requery/kotlin/repository/sample/SampleConfig.kt
package org.springframework.data.requery.repository.sample

import org.springframework.context.annotation.Configuration
import org.springframework.data.requery.kotlin.configs.RequeryTestConfiguration
import org.springframework.data.requery.kotlin.repository.config.EnableRequeryRepositories

/**
 * org.springframework.data.requery.repository.sample.SampleConfig
 *
 * @author debop
 */
@Configuration
@EnableRequeryRepositories
class SampleConfig: RequeryTestConfiguration()